Source/Shared/BPConfiguration.m

```diff
@@ -70,6 +70,10 @@
         "Exclude a testcase in the set of tests to run (takes priority over `include`)."},
     {'X', "xcode-path", required_argument, NULL, BP_VALUE | BP_PATH, "xcodePath",
         "Path to xcode."},
+    {'u', "use-simulator", required_argument, NULL, BP_VALUE, "useDeviceID",
```

Comment:
Let's see from a different angle. From the bluepill driver's perspective, this is device to REUSE. But from the bp's perspective, this is just a device specified from CLI. It is the first use, or second use, it doesn't know and doesn't care. Since this option is for bp use only, so I prefer the variable name "use" to "reuse".